src/Student.js

```diff
@@ -42,7 +42,13 @@ function GenerateStudentBody(user)
     });
     
     body += "<p>Total: &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;" + user.GetFinalGrade().toFixed(2) + "/100</p>";
-    body += "<p>Letter Grade:&emsp;&emsp;&emsp;" + user.GetLetterGrade() + "</p>"; 
+    if (typeof user.GetLetterGrade() === "undefined"){
```

Comment:
This part of the code is not necessary as we do not allow a letter grade to be undefined. The equivalent of a missing letter grade is the string "Not Submitted Yet" which can simply be sent to the web page directly